tests/test_idpi/fieldextra_templates/test_regrid.nl

```diff
@@ -0,0 +1,70 @@
+&RunSpecification
+ additional_profiling=.true.
+ strict_nl_parsing=.true.
+ verbosity="moderate"
+ diagnostic_length=110
+ soft_memory_limit=350.0
+ wait_time=10
+ mx_wait_time=14400
+ n_ompthread_total=1
+ n_ompthread_collect=1
+ n_ompthread_generate=1,1
+ out_cost_expensive=1000
+/
+&GlobalResource
+ dictionary="/project/s83c/fieldextra/tsa/resources/dictionary_cosmo.txt"
+ grib_definition_path="/project/s83c/fieldextra/tsa/resources/eccodes_definitions_cosmo",
+                        "/project/s83c/fieldextra/tsa/resources/eccodes_definitions_vendor"
+ grib2_sample="/project/s83c/fieldextra/tsa/resources/eccodes_samples/COSMO_GRIB2_default.tmpl"
+ rttov_coefs_path="/project/s83c/fieldextra/tsa/resources/rttov_coefficients"
```

Comment:
There's no /project on Balfrin.
If this code needs to run on Balfrin eventually, we should look for a better solution. If not, fine with me.